.codesandbox/tasks.json

```diff
@@ -0,0 +1,18 @@
+{
+    "setupTasks": [
+      {
+        "name": "Install Dependencies",
+        "command": "npm install"
+      }
+    ],
+    // Adding Start task
+    "tasks": {
+      "npm install; npm start": {
+        "name": "Start",
+        "command": "npm install; npm start",
+        "preview": {
+          "port": 3000
+        }
+      }
+    }
+  }
```

Comment:
The file should end in a new line to be consistent with the rest of the repo

Reply:
Sorry, I fixed that.